docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,696 @@
+  --colors-primary-900: 23 64 49;
+}
+```
+::
+
+::info
+For more information on using CSS variables with Tailwind, see the [Tailwind CSS documentation](https://v3.tailwindcss.com/docs/customizing-colors#using-css-variables).
+::
+
+## Setting Up the Storefront
+
+Now you need to configure your Storefront to send the appropriate config ID to the Middleware based on the current URL path.
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+In your Next.js SDK configuration, implement the `getConfigSwitcherHeader` function:
+
+```ts
+// apps/storefront-unified-nextjs/sdk/config.ts
+import { defineSdkConfig, getPathnameFromRequestHeaders } from '@vue-storefront/next';
+import type { UnifiedEndpoints } from 'storefront-middleware/types';
+
+function getConfigSwitcherHeader(baseHeaders: Record<string, string>) {
+  const pathname =
+    typeof window !== 'undefined' ? window.location.pathname : getPathnameFromRequestHeaders(baseHeaders);
+  
+  // Determine the config ID based on URL path
+  return pathname?.includes('/electronics') ? 'electronics' : 'apparel';
+}
+
+export function getSdkConfig() {
+  return defineSdkConfig(({ buildModule, config, getRequestHeaders, middlewareModule }) => ({
+    unified: buildModule(middlewareModule<UnifiedEndpoints>, {
+      apiUrl: `${config.apiUrl}/commerce/unified`,
+      defaultRequestConfig: {
+        getConfigSwitcherHeader,
+        headers: getRequestHeaders,
+      },
+      // ... other config
+    }),
+    // ... other modules
+  }));
+}
+```
+
+#tab-2
+In your Nuxt SDK configuration, define a reusable `getConfigSwitcherHeader` function:
+
+```ts
+// apps/storefront-unified-nuxt/sdk.config.ts
+import type { CommerceEndpoints, UnifiedCmsEndpoints, UnifiedEndpoints } from 'storefront-middleware/types';
+
+export default defineSdkConfig(({ buildModule, config, getRequestHeaders, middlewareModule }) => {
+  const route = useRoute();
+  
+  // Create a reusable function for determining the config ID
+  function getConfigSwitcherHeader() {
+    return route.path.includes('/electronics') ? 'electronics' : 'apparel';
+  }
+  
+  return {
+    // Unified commerce module
+    unified: buildModule(middlewareModule<UnifiedEndpoints>, {
+      apiUrl: `${config.apiUrl}/commerce/unified`,
+      defaultRequestConfig: {
+        getConfigSwitcherHeader,
+        headers: getRequestHeaders(),
+      },
+      // ... other config
+    }),
+    // ... other modules
+  };
+});
+```
+::
+
+## Updating the File Structure for Path Routing
+
+To enable path-based routing, you need to restructure your application to include a dynamic `[configId]` parameter in the route. This allows you to access the store identifier directly from the URL.
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+For Next.js apps using the App Router, update your file structure:
+
+```bash
+apps/storefront-unified-nextjs/app/
+├── [locale]/                 # Language parameter
+│   ├── [configId]/           # Store identifier parameter
```

Comment:
Do we have anything against having the order reversed? If no, then please add an information that this can go both ways

Reply:
it didn't work in Next.js - at least without any other changes in the i18n config

Reply:
Ty, I've created a spike for the next sprint, unfortunatelly there's a business case for this